pulsar-io/kafka/pom.xml

```diff
@@ -62,6 +62,23 @@
         <groupId>org.apache.nifi</groupId>
         <artifactId>nifi-nar-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <version>${spotbugs-maven-plugin.version}</version>
```

Comment:
The version is already defined in the top level pom, this will print a maven warning.

Reply:
done.